embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/prefs/PreferencesService.kt

```diff
@@ -85,6 +85,13 @@ interface PreferencesService {
      */
     fun incrementAndGetNativeCrashNumber(): Int
 
+    /**
+     * Increments and returns the AEI crash number ordinal. This is an integer that
+     * increments on every NDK tombstone detected via an AEI trace. It allows us to check the % of AEI crashes that
+     * didn't get delivered to the backend.
+     */
+    fun incrementAndGetAeiCrashNumber(): Int
```

Comment:
should all of these be named `AeiNativeCrash` instead of `AeiCrash`? I undersatnd AEI can report both native and java crashes, right? I know we are not doing anything with the latter for now, but we may in the future, so it's good to distinguish?

Reply:
[AEI](https://developer.android.com/reference/android/app/ApplicationExitInfo#getTraceInputStream()) only contains a trace for ANRs and native crashes. While it does record that a JVM crash happened, it doesn't record the stacktrace. The SDK should only send an AEI with `REASON_CRASH` if an ANR trace is associated with it.